viahtml/blocklist.py

```diff
@@ -0,0 +1,116 @@
+"""The abstract blocklist object."""
+
+import os
+import re
+from enum import Enum
+from logging import getLogger
+from urllib.parse import urlparse
+
+
+class Blocklist:
+    """A blocklist which URLs can be checked against.
+
+    The blocklist is stored at:
+      * https://hypothesis-via.s3-us-west-1.amazonaws.com/via-blocklist.txt
```

Comment:
Maybe just say it's stored at the location given by the envvar? I'm not sure we want to publish this URL. Plus we might change it or make it different for different envs

Reply:
Yeah, I think it needs to go